WordPress/WordPressTest/RemoteTestCase.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import XCTest
+import OHHTTPStubs
+@testable import WordPress
+
+class RemoteTestCase: XCTestCase {
+
+    // MARK: - Constants
+
+    let contentTypeJson     = "application/json"
+    let contentTypeJS       = "text/javascript;charset=utf-8"
+    let contentTypeHTML     = "application/html"
+    let timeout             = TimeInterval(1000)
+
+    // MARK: - Properties
+
+    var restApi: WordPressComRestApi!
+    var contextManager: TestContextManager!
+
+    // MARK: - Overridden Methods
+
+    override func setUp() {
+        super.setUp()
+        contextManager = TestContextManager()
+        restApi = WordPressComRestApi(oAuthToken: nil, userAgent: nil)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+
+        ContextManager.overrideSharedInstance(nil)
+        contextManager.mainContext.reset()
+        contextManager = nil
+        restApi = nil
+        OHHTTPStubs.removeAllStubs()
+    }
+
+    // MARK: - Helpers
+
+    func stubRemoteResponse(_ endpoint: String, filename: String, contentType: String, status: Int32 = 200) {
```

Comment:
It doesn't HAVE to but consider putting this into an extension for `RemoteTestCase`.